NSSNamespace/User_Authentication
# Ew payment view

## Changelog
- Pulled over ```Views/Payment/Create.cshtml``` from old project
- Adjusted namespace reference accordingly

## To Test
- ```git fetch```
- ```git checkout ew-payment-view```
- run app
- once your browser has navigated to the **Bangazon!** home page, type http://localhost:5000/payment/create into navigation bar, or click link provided
- verify payment form is displayed
